docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc

```diff
@@ -1,18 +1,16 @@
 [id="configuring-networking-for-control-plane-services_{context}"]
 
-= Configuring the networking for control plane services
+= Configuring isolated networks on control plane services
 
-Once NMState operator created the desired hypervisor network configuration for
-isolated networks, we need to configure {rhos_prev_long} ({OpenStackShort}) services to use configured
-interfaces. This is achieved by defining `NetworkAttachmentDefinition` custom resources (CRs) for
-each isolated network. (In some clusters, these CRs are managed by the Cluster
-Network Operator in which case `Network` CRs should be used instead. For more information, see
-link:https://docs.openshift.com/container-platform/4.15/networking/cluster-network-operator.html[Cluster
-Network Operator] in _OpenShift Container Platform 4.15 Documentation_.)
+After the NMState operator creates the desired hypervisor network configuration for isolated networks, you must configure the {rhos_prev_long} ({OpenStackShort}) services to use the configured interfaces. You define a `NetworkAttachmentDefinition` custom resource (CR) for each isolated network. In some clusters, these CRs are managed by the Cluster Network Operator, in which case you use `Network` CRs instead. For more information, see
```

Comment:
I'd advise to check if "in some clusters" is applicable for downstream. I'd think we have a particular way of deploying our OpenShift cluster. Do we indeed instruct users to consider cluster-network-operator as well as the alternative?

---

Note that we probably still want to point to the possibility for upstream sake, I just am not sure if the alternatives are possible for RHOSO. (I can try to clarify this if needed. Let me know.)

Reply:
@booxter Do you know who we can ask about the cluster-network-operator? I can conditionalize the text for upstream only for now.